etc/profile-a-l/luarocks.profile

```diff
@@ -0,0 +1,74 @@
+# Firejail profile for luarocks
+# Description: LuaRocks is the package manager for the Lua programming language.
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include luarocks.local
+# Persistent global definitions
+include globals.local
+
+# disable blacklist for lua interpreter paths
+noblacklist ${PATH}/lua*
+noblacklist /usr/include/lua*
+noblacklist /usr/lib/liblua*
+noblacklist /usr/lib/lua
+noblacklist /usr/lib64/liblua*
+noblacklist /usr/lib64/lua
+noblacklist /usr/share/lua*
+
+include disable-common.inc
+# luarocks can invoke compilers
+#include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+# luarocks is hacky and needs shell access
+#include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+allow ${HOME}/.netrc
+allow ${HOME}/.config/pkcs11
+allow ${HOME}/.wget-hsts
+allow ${HOME}/.cache/luarocks
+allow ${HOME}/luarocks/cmd/external
+allow ${HOME}/.nix-profile/bin
+allow ${HOME}/.luarocks
+allow ${HOME}/.config/luarocks
+
+allow /usr/share/ca-certificates
+allow /usr/share/p11-kit
+allow /usr/share/terminfo
+allow /usr/share/lua
```

Comment:
```suggestion
whitelist /usr/share/lua
include whitelist-usr-share-common.inc
```

Reply:
`whitelist-usr-share-common.inc` includes stuff like `/usr/share/alsa` or `/usr/share/texlive`. 
Is the reasoning here that luarocks is never be run as root, so accessing /usr/share is fine asit is only exposes relative unuseful information for software blocked elsewhere?

Reply:
The reason is that `whitelist-*-common.inc` is well tested across different systems/configurations/hardware/software/distributions and simplifies maintenance.

This means that
1. If we get tomorrow a report that PoloniumLinux requires `/usr/share/pold` for dynamic linking and `/usr/share/tls` for TLS/HTTPS, all we need is to add two lines into wusc instead of search for all possible affected profiles and adding these lines in them.
2. New profiles (like luarocks) work across different systems/configurations/hardware/software/distributions. Did you tested luarocks with this minimal wusc across different systems/configurations/hardware/software/distributions? Did you considered systems which use `/usr/share/crypto-policies` for (TLS-)certificates? Did you considered lua-packages with "native-extensions" (IDK how lua calls them) that use perl for building? As an example, if I want to build a rust binary that statically links against openssl, I need perl. I assume there a similar cases with lua. (Yes I know that perl is blacklisted anyway). What is if a packages wants to `file ./foobar`?